src/providers/aad/index.ts

```diff
@@ -0,0 +1,17 @@
+import { assignDefaults, addAuthorize } from '../../utils/provider'
+
+export default function aad (nuxt, strategy) {
+  assignDefaults(strategy, {
+    scheme: 'oauth2',
+    endpoints: {
+      authorization: `https://login.microsoftonline.com/${strategy.tenantId}/oauth2/v2.0/authorize`,
+      userInfo: 'https://graph.microsoft.com/v1.0/me',
+      token: `https://login.microsoftonline.com/${strategy.tenantId}/oauth2/v2.0/token`
```

Comment:
```suggestion
      authorization: `https://login.microsoftonline.com/${strategy.tenantId || 'common'}/oauth2/v2.0/authorize`,
      userInfo: 'https://graph.microsoft.com/v1.0/me',
      token: `https://login.microsoftonline.com/${strategy.tenantId || 'common'}/oauth2/v2.0/token`
```

Does this make sense @JoaoPedroAS51 @bmulholland ?

Reply:
I think should be like this:

```js
  const tenantId = strategy.tenantId || 'common'

  assignDefaults(strategy, {
    scheme: 'oauth2',
    endpoints: {
      authorization: `https://login.microsoftonline.com/${tenantId}/oauth2/v2.0/authorize`,
      userInfo: 'https://graph.microsoft.com/v1.0/me',
      token: `https://login.microsoftonline.com/${tenantId}/oauth2/v2.0/token`
    },
    codeChallengeMethod: 'S256',
    scope: ['openid', 'profile'],
    autoLogout: true
  })
```

Reply:
Agreed, @JoaoPedroAS51's approach is much easier to read. I'd suggest a comment above the `|| "common"` line that says this:
```
// Allows users with both personal Microsoft accounts and work/school accounts from Azure AD to sign into the application.
// https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-v2-protocols#endpoints
```